### ISSUE TITLE
New journey map

### ISSUE THREAD

**Issue by Konrad22** [cooljeanius/The_Earths_Gut#39]
To replace the old wesnoth map.

**Comment by cooljeanius** [comment]
Will require me to figure out how to either revive trackplacer, or use that Tiled plugin...

**Comment by Konrad22** [comment]
Just tracking it for now.

**Comment by cooljeanius** [comment]
I think @babaissarkar was working on reviving Trackplacer recently?

**Comment by babaissarkar** [comment]
> I think [@babaissarkar](https://github.com/babaissarkar) was working on reviving Trackplacer recently?

Yeah, the current version can be obtained here: https://github.com/babaissarkar/WesnothMapTracker
